src/views/account-details/account-details.view.jsx

```diff
@@ -359,7 +361,7 @@ const mapStateToProps = (state) => ({
 });
 
 const mapDispatchToProps = (dispatch) => ({
-  onLoadAccount: (accountIndex) => dispatch(accountDetailsThunks.fetchAccount(accountIndex)),
+  onLoadAccount: (accountIndex, fiatExchangeRates, preferredCurrency) => dispatch(accountDetailsThunks.fetchAccount(accountIndex, fiatExchangeRates, preferredCurrency)),
```

Comment:
Why sending all this extra data instead of picking them from `redux` in the thunks? I think that this is makes views more verbose.

Reply:
I made this decision for 2 reasons:

 * In this case, the view already has the props `fiatExchangeRates` and `preferredCurrency` so no new props need to be added to the view. 
 * If we access `fiatExchangeRates` from redux in the thunk, we need to test again the state of the async task before accessing its data.

Also, and this is subjective, I prefer functions to operate on their input parameters rather than accessing the state. It feels more pure and deterministic.

Anyway, if this does not fit with the style of the application I can change it. Should I?

Reply:
Nope, it's perfectly fine. Just wanted to know the reason! I totally agree with this also: `I prefer functions to operate on their input parameters rather than accessing the state. It feels more pure and deterministic.`